packages/garbo/src/lib.ts

```diff
@@ -383,7 +382,7 @@ export function pillkeeperOpportunityCost(): number {
   const alternateUses = [
     {
       can: canTreasury,
-      value: EMBEZZLER_MULTIPLIER() * get("valueOfAdventure"),
+      value: TARGET_MULTIPLIER() * get("valueOfAdventure"),
```

Comment:
This replace of Embezzler feels wrong under the current paradigm given this is specifically accounting for targetting embezzlers via lucky?

Or is the (future) intent to include embezzler calcs within target_multiplier()?